### ISSUE TITLE
[BUG] TF2BD opens and exhausts all ports on launch screen

### ISSUE THREAD

**Issue by luckydonald** [surepy/tf2_bot_detector#25]
**Describe the bug**
> 

**To Reproduce**
1. Start bot detector
2. Launch TF2 from bot detector
3. play some tf2 but be frustrated by 9 bots on your own team and disconnect from servers a lot.
4. Close TF2;  
    Bot detector now looks like this  
    ![image](https://github.com/surepy/tf2_bot_detector/assets/2737108/9e5015c9-458d-4d4f-8ec9-e12f375325f8)
5. Wait.

Now:
- A) Wait about 10 hours, and you can't open google.com in your webbrowser, as bot detector will eventually saturate all 65535 local ports.
- B) This can be confirmed by Powershell's `Get-NetTCPConnection`, 
- C) Or by looking at the Handles count in Process Informer > Processes > tf2_bot_detector.exe > Properties > Statistics > Other > Peak Handles raising about every 5 seconds. Repeat Step 2 and 3 to make it more obvious.

> A clear and concise description of what you expected to happen.

**Logs**
> Add any relevant logs. Logs are found by going to File → Open Logs Folder

**Screenshots**
> Powershell > `Get-NetTCPConnection`
> ![2024-01-11 00_32_59-System Informer  LUCKYDONALD-PC_luckydonald](https://github.com/surepy/tf2_bot_detector/assets/2737108/8bc434f5-c7e6-49ed-909d-98cf0d609a24)

**Desktop (please complete the following information):**
> - OS: Windows 10
> - Version of bot detector: 1.5.12, but i didn't see anything relevant in the changelogs about this in the newer 1.5.13.

**Additional context**
> My guess is that it is trying to connect (RCON?) to tf2 to see if it's (already/again) launched, but doesn't properly clean up the socket connections after each test, so it's opening them but never closing.
> if you keep the program open for a day or more as you ignore it in the background, it will use up more and more ports until eventually other programs can't connect to shit no more.

**Comment by surepy** [comment]
holy shit; okay, confirmed.

```
PS C:\Users\sleepy> Get-NetTCPConnection | Select LocalPort, OwningProcess | findstr 34092
    (list grows...)
    53868         34092
    53867         34092
    53866         34092
    53865         34092
    53863         34092
    53861         34092
    53859         34092
    53858         34092
    53857         34092
    53856         34092
    53855         34092
    53848         34092
    53846         34092
    53845         34092
    53838         34092
    53831         34092
    53830         34092
    53829         34092
    53828         34092
    53827         34092
    53826         34092
    53825         34092
    53822         34092
    53820         34092
    53819         34092
    53817         34092
    53813         34092
    53812         34092
    53811         34092
    53808         34092
    53807         34092
    53806         34092
    53805         34092
    53803         34092
    53868         34092
    53825         34092
    53822         34092
 ```

**Comment by surepy** [comment]
i will release soontm

from my limited testing there will be some ports lingering around for a bit after stopping tf2 but they'll eventually close (and definitely not create more open ports)

**Comment by luckydonald** [comment]
hey, if it's 2000 open ports, I don't care. It's only getting annoying if it takes ALL of them.

Thanks for the fix.

**Comment by surepy** [comment]
soontm = 1 month

anyway released in 1.5.14
